neqo-transport/src/cid.rs

```diff
@@ -312,7 +304,7 @@ impl ConnectionIdEntry<[u8; 16]> {
         builder.encode_varint(self.seqno);
         builder.encode_varint(0u64);
         builder.encode_vec(1, &self.cid);
-        builder.encode(&self.srt);
+        builder.encode(self.srt.as_bytes());
```

Comment:
If `builder.encode` were to take `impl AsRef<[u8]>`, this wouldn't have needed to change.